README.md

```diff
@@ -87,7 +87,28 @@ The most important changes are:
   sequence to differentiate conflicting slug, but rather a UUID (e.g. something
   like `2bc08962-b3dd-4f29-b2e6-244710c86106`). This makes the
   codebase simpler and more reliable when running concurrently, at the expense
-  of uglier ids being generated when there are conflicts.
+  of uglier ids being generated when there are conflicts. 
+* If you need previously used numeric sequence for some reason:
```

Comment:
should we consider re-framing this sentence to "If you want to override UUID with numeric sequence: "
or
add a sentence like "the old behaviour is unsupported by the new code base and discouraged, but if you really think you need it check issues #430 #480, #576 and #602 for discussion and workarounds". as mentioned in #602 (comment) 
or 
do nothing :smile: